common/teacmd/git.go

```diff
@@ -3,11 +3,27 @@ package teacmd
 import (
 	"bufio"
 	"bytes"
+	"crypto/rand"
+	"encoding/hex"
 	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/magefile/mage/sh"
+	toml "github.com/pelletier/go-toml/v2"
+	"github.com/rotisserie/eris"
+)
+
+const (
+	oldModuleName = "github.com/argus-labs/starter-game-template/cardinal"
+
+	tomlFile = "world.toml"
+
+	routerKey = "ROUTER_KEY"
+
+	tomlSection = "common"
```

Comment:
this should prob be a bit more specific like `tomlSectionCommon`

Reply:
updated